specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/virtualNetworkGateway.json

```diff
@@ -1825,6 +1825,63 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/connections/{virtualNetworkGatewayConnectionName}/getikesas": {
```

Comment:
Please name the action "listIkeSas" (if IKE is something people understand).  The "list" prefix is more common and is in line with the description of the API.

Reply:
Hi @pilor , the action "getikesas" is the one that was approved by the Program Manager [Yu-Shun Wang (yu-shun.wang@microsoft.com)]

Reply:
Please work with them to change the name to something without the prefix "get".  "Get" is overloaded as it is also an HTTP verb and shouldn't be used in this manner in a REST API.  "list" is fine or whatever else you come up with